linode/manager
# Disable Mongo integration tests

## Description 📝

**What does this PR do?**
Disables Mongo tests in Cypress.

## How to test 🧪

**What are the steps to reproduce the issue or verify the changes?**
Run `yarn up`, then:
```
yarn cy:run -s "cypress/e2e/databases/create-database.spec.ts,cypress/e2e/databases/update-database.spec.ts"
```

Confirm that the Mongo tests are disabled (in other words, that the tests pass).
